DatadogLogs/Sources/Feature/MessageReceivers.swift

```diff
@@ -152,6 +152,27 @@ internal struct CrashLogReceiver: FeatureMessageReceiver {
         let carrierInfo: CarrierInfo?
         /// Current user information.
         let userInfo: UserInfo?
+
+        /// A type representing part of last RUM view information required to link crash log with previous RUM session.
+        /// It mirrors the schema of `RUMViewEvent`, so we can decode it from the last `RUMViewEvent` coded in crash context.
+        struct PartialRUMViewEvent: Decodable {
+            struct Application: Decodable {
+                let id: String
+            }
+            struct Session: Decodable {
+                let id: String
+            }
+            struct View: Decodable {
+                let id: String
+            }
+
+            let application: Application
+            let session: Session
+            let view: View
+        }
```

Comment:
Is there any public/internal doc explaining how this correlation work? In case when there an update needed we can always refer.

Also, we have been doing quite a few changes to make the correlation work and so far they have been very organic.  These things impact SDK and we should document them somewhere.

Reply:
Fair call 👍. I took an extra time to document logs, spans and RUM correlations in [this spec (internal)](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/3416820413/Event+Correlations+-+SDK+Datadog+App+Contracts).